srv/GetEntitiesStates.srv

```diff
@@ -1,5 +1,6 @@
 # Get objects in the scene which can be interacted, e.g. with using SetEntityState.
 # Use GetEntities service instead if EntityState information for all entities is not needed.
+# Support for this interface is indicated through the ENTITY_STATE_LISTING value in GetSimulationFeatures.
```

Comment:
ENTITY_STATE_GETTING (according to the new naming)

Reply:
Nice catch, thanks!